doc/advanced-topics.md

```diff
@@ -127,6 +127,58 @@ Other unrecognised `type` attributes will be passed as NGSI-LD data using the fo
     }
```

Comment:
Maybe the support to linked data is a relevant item to mention in the CHANGES_NEXT_RELEASE sublist at https://github.com/telefonicaid/iotagent-node-lib/blob/master/CHANGES_NEXT_RELEASE#L7-L14 ?

Reply:
Fixed 891e514